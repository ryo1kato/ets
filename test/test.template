var1=${VAR1}
var2=${VAR2}
var3=${VAR3}
var4=${VAR4}

var5=${VAR5}
var6=${VAR6}
var7=${VAR7}
var8=${VAR8}

var9=${VAR5}
var10=${VAR10}
var11=${VAR11}
var12=${VAR12}

-- heredoc 1 --
${HEREDOC1}
-- heredoc 2 --
${HEREDOC2}
-- heredoc 3 --
${HEREDOC3}
