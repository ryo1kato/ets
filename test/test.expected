var1=val1
var2=val2
var3=val3
var4=val4

var5=val5
var6=val6
var7=val7
var8=val8

var9=val5
var10= val10
var11=val11 # this is not comment
var12=val12"

-- heredoc 1 --
this is a test
of heredoc
number 1

-- heredoc 2 --
    this is a test
    of heredoc
    number 2

-- heredoc 3 --
1
 2
  3
   4
    5
     6
      7

---------------
